doc/object_properties.adoc

```diff
@@ -0,0 +1,252 @@
+* "mesh": 1 texture for each mesh material
+* "sprite": 1 texture
+* "upright_sprite": 2 textures
+
+=== `colors`
+* field type: `ColorSpec`
+
+WARNING: This feature is not functional.
+
+=== `spritediv`
+* field type: `2d_vector`
+
+Determines the number of columns, rows in a spritesheet texture, which is used for animations.. Wanted value is `{x = columns, y = rows}`.
+
+=== `initial_sprite_basepos`
+* field type: `2d_vector`
+
+Determines the position of the first frame to use in the spritesheet texture. Wanted value is `{x = column, y = row}`.
+
+=== `is_visible`
+* field type: `bool`
+
+Whether the object is visible and can be pointed at.
+
+=== `makes_footstep_sound`
+* field type: `bool`
+
+Whether the object makes a footstep sound when it steps on nodes that that have footstep sounds.
+
+=== `stepheight`
+* internal type: `f32`
+
+How high in nodes the object can step up.
+
+=== `automatic_rotate`
+* internal type: `f32`
+
+How high in nodes the object can step up.
+
+=== `automatic_face_movement_dir`
+* field type: `int` or `false`
+* unit: `degrees`
+
+TIP: This property can be set to `false` to disable it.
+
+Make the object automatically face the direction it is moving in, the value is the offset in degrees.
+
+=== `backface_culling`
+* field type: `bool`
+
+Whether or not a model is backface culled.
+
+=== `glow`
+* field type: `int`
+
+=== `nametag`
+* field type: `string`
+
+Text displayed about an object, empty by default for all objects except players, in which case, it is their name.
+
+=== `nametag_color`
+* internal type: `ColorSpec`
+
+The color that the nametag text will be.
+TIP: This property can be set to `false` to disable it.
+
+=== `nametag_bgcolor`
+* field type: `color_def` or `false`
+* internal type: `ColorSpec`
+
+TIP: This property can be set to `false` to disable it.
+
+=== `automatic_face_movement_max_rotation_per_sec`
+* field type: `int`
+
+Set the maximum automatic rotation to this value, no limit is the value is less than or equal to 0.
+
+=== `infotext`
+* field type: `string`
+
+When a player points at the object, they will be shown the text.
+
+=== `static_save`
+* field type: `bool`
+
+Whether the object is saved  statically, if not, it will be deleted when the world block is unloaded.
+
+=== `wield_item`
+* internal type: `ItemString`
+
+Only when using "wielditem" visual
+
+=== `use_texture_alpha`
+* field type: `bool`
+
+Whether or not the texture's alpha should be used.
+
+=== `shaded`
+* field type: `bool`
+
+Whether or not diffuse lighting is enabled for the object.
+
+=== `show_on_minimap`
+* field type: `bool`
+
+Whether or not the object is visible on the minimap.
+
+=== `damage_texture_modifier`
+* field type: `string`
+
+A texture modifier that will be appended to the object's current textures for the duration of the damage flash when the object is damaged.
+
+== Layout of Object Properties
+
+The way these properties are laid out is in a table, the order of which is not important.
```

Comment:
Remove the second part of the sentence. Order of the hash part is never important in Lua - this belongs somewhere else, in a Lua primer maybe. Consider noting that extraneous entries will be ignored, but might be interpreted incorrectly by future Minetest versions (which is why they should be prefixed with an underscore). Also please note that this uses the regular indexing, so metatables work as expected.